00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,542 @@
+- done: all issues and PRs go here when done or merged, respectively
+
+Here's a quick diagram:
+
+![Complete issue workflow diagram](./project-board.svg)
+
+In more technical terms, this would be an organization-level Github project
+board. This will allow for issues and PRs across _any_ Enarx repository (not
+just the main one) to be included, which supports modularization of the project
+as it grows.
+
+## Meetings
+
+The concepts and processes introduced above are put into action in the form of
+several team meetings that happen throughout the process of a single sprint.
+
+To start, consider this high-level overview of how an issue progresses through
+the process, beginning with its opening and continuing all the way until it's
+considered done:
+
+![Complete issue workflow diagram](./full-process.svg)
+
+This flowchart introduces the three crucial events of the process, each of
+which is encapsulated by its own team meeting:
+ 
+ - **Planning**, a meeting representing the start of a new sprint
+ - **Daily Standup**, the team's daily meeting where, among other things,
+   **Triage** is performed.
+ - **Retrospective**, or "Retro" for short, a meeting representing the
+   end of a sprint
+
+Once an issue is filed, it will be categorized and discussed in Triage. If
+urgent, it's injected straight into the current sprint; if not, it gets pushed
+to the backlog. From there, the rest of the cycle will carry it through to
+completion: Sprint Planning will sooner or later land it as part of a sprint,
+where it will become a team member's action item to complete. At the end of the
+sprint, Sprint Retro will allow the team to acknowledge its completion or need
+for extra time as appropriate. When finished, the issue is considered complete.
+
+The following sections will dive deep into the process of each meeting,
+carefully considering actions that need to be taken, who should be responsible
+for those actions, and how automation might be able to aid those actions.
+
+### Planning
+
+Sprint planning constitutes the beginning of a new sprint. It's where the team
+and project leadership collectively decide what the project's focus should be.
+
+The sprint planning process is illustrated in the following flowchart:
+
+![Sprint Planning Diagram](./sprint-planning.svg)
+
+Note that sprint planning only considers issues, not pull requests.
+
+The meeting will start with a triage, as described above. This exists to ensure
+no issues are uncategorized before the planning step. (This is necessary since
+sprint planning will supersede the day's standup and, consequently, its triage,
+as described later.)
+
+With the backlog fully up to date after the triage, the entire team will take
+a small amount of time to **nominate** issues from the area they work on that
+they think are most compelling. It should be each individual team member's
+responsibility to nominate issues of interest _before_ planning begins;
```

Comment:
This is a bit buried, but I think is very important so that we don't get bogged down in going through every single backlog issue during nominations or have to wait while people make decisions. An inefficient process here would cut into our time for targeted discussion or make it likely that the meeting runs over-time.

In fact, I would argue that during triage, we should have the choice of moving non-urgent issues either to backlog or, if likely to be a priority for next sprint, then straight to "nominated"; this allows us to speed up our individual pre-planning nomination process by being able to quickly throw a small amount of issues from "nominated" into backlog if we want to de-prioritize them, rather than combing through a large amount of backlog issues to remember what we wanted to add to "nominated." 

If the above idea is too unorthodox for sprints, at the very least I think the responsibility described here that the development team moves issues into "nominated" before planning should be a semi-hard requirement. Last minute-nominations can make sense, but they should be the exception and the time limit on this should be enforced.

This way we have adequate time to discuss all the nominations and we minimize (individually and as a team) the process of searching through dozens of backlog issues to find a specific issue we know we want to nominate (but may have forgotten the exact wording of the title, etc.) while preparing for sprints.

Reply:
I personally like that idea. Maybe it can be a new state in the triage process flowchart if the issue is decided to be "not urgent".

Reply:
I didn't see where / if this was addressed, but maybe since it's a small detail it makes more sense to discuss as a team rather than put it in the document. Especially as I don't know others' thoughts on it.